### ISSUE TITLE
fragment not recognised, or get message never used :)

### ISSUE THREAD

**Issue by maapteh** [dotansimha/graphql-eslint#202]
Im dumping log here, will write more details later

```
/Users/mpth/Git/snip/web/src/common/fragments/cook-book-user.graphql
  1:2  error  [NoUnusedFragments] Fragment "CookBookUser" is never used  @graphql-eslint/validate-against-schema

/Users/mpth/Git/snip/web/src/common/fragments/cook-book.graphql
  1:2   error    [NoUnusedFragments] Fragment "CookBook" is never used                                                             @graphql-eslint/validate-against-schema
  1:31  warning  Field "id" must be selected when it's available on a type. Please make sure to include it in your selection set!  @graphql-eslint/require-id-when-available
  3:10  warning  Field "id" must be selected when it's available on a type. Please make sure to include it in your selection set!  @graphql-eslint/require-id-when-available
  4:13  error    [KnownFragmentNames] Unknown fragment "CookBookUser"  
```

So one file its saying not used, then the other is complaining. I added my schema and all operations to the eslint config. Hopefully correct will debug more tomorrow

**Comment by dotansimha** [comment]
2 different issue. see: 
- https://github.com/dotansimha/graphql-eslint/issues/219 
- https://github.com/dotansimha/graphql-eslint/issues/89
